users/models.py

```diff
@@ -44,3 +44,6 @@ class User(AbstractUser):
     objects = UserManager()
     username: None = None
     email: str = models.EmailField(unique=True)
+
+    def get_saved_posts(self: "User") -> models.QuerySet:
```

Comment:
Architecture wise, it would be better to have this method on SavedPost model. This way user model does not know anything about SavedPost, which is good because we can easily remove savedPost or copy user model to new project. 
So please move this method into SavedPost

Reply:
You missed this one.